eng/common/scripts/Package-Properties.ps1

```diff
@@ -1,14 +1,13 @@
 # Helper functions for retireving useful information from azure-sdk-for-* repo
-# Example Use : Import-Module .\eng\common\scripts\modules
 class PackageProps
 {
-    [string]$pkgName
-    [string]$pkgVersion
-    [string]$pkgDirectoryPath
-    [string]$pkgServiceName
-    [string]$pkgReadMePath
-    [string]$pkgChangeLogPath
-    [string]$pkgGroup
+    [string]$Name
+    [string]$Version
+    [string]$DirectoryPath
+    [string]$ServiceName
```

Comment:
Sorry I've been adding comments in various language PR's as I see things and I realize it might be hard to track everything (it definitely is for me :)).

So to move my feedback here I think we should update ServiceName to ServiceDirectory throughout all these scripts to be consistent with the rest of our pipelines as well as avoid confusion with having this be a nice display name, which we also have in some places, like here https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/update-docs-metadata.ps1#L64 which is actually a nice display name as opposed to the directory it lives in.